nengo_bones/tests/test_check_deploy.py

```diff
@@ -131,10 +131,8 @@ def test_subject(tmp_path, monkeypatch):
 def test_pypirc(tmp_path, monkeypatch):
     _monkeypatch(tmp_path, monkeypatch)
     result = _write_repo_and_check(tmp_path, pypirc=False)
-    assert_exit(result, 1)
```

Comment:
We should test both cases (pypirc exists and does not exists). You can make a mock pypirc file to test the case where it exists.